app/engines/bing_image_response_parser.rb

```diff
@@ -1,4 +1,4 @@
-class BingV6ImageResponseParser < BingV6ResponseParser
+class BingImageResponseParser < BingResponseParser
   def results
```

Comment:
BingV6ImageResponseParser is also being referenced in : coverage/index.html line 43958 and 44265